src/EFCore.PG/Extensions/NpgsqlModelBuilderExtensions.cs

```diff
@@ -247,6 +247,93 @@ public static class NpgsqlModelBuilderExtensions
 
         #endregion
 
+        #region Composite
+
+        /// <summary>
+        /// Registers a user-defined composite type in the model.
+        /// </summary>
+        /// <param name="modelBuilder">The model builder in which to create the composite type.</param>
+        /// <param name="schema">The schema in which to create the composite type.</param>
+        /// <param name="name">The name of the composite type to create.</param>
+        /// <param name="fields">The composite list of fields, with their names and PostgreSQL types.</param>
+        /// <returns>
+        /// The updated <see cref="ModelBuilder"/>.
+        /// </returns>
+        /// <remarks>
+        /// See: https://www.postgresql.org/docs/current/rowtypes.html
+        /// </remarks>
+        /// <exception cref="ArgumentNullException">builder</exception>
```

Comment:
`builder` isn't a parameter. Could you change this to a `<paramref name="modelBuilder" />`?